consumer/tests/__init__.py

```diff
@@ -80,54 +79,43 @@ def first(path, obj):
 
 @pytest.mark.unit
 @pytest.fixture(scope='session')
-def birdisle_server():
-    password = config.get_consumer_config().get('REDIS_PASSWORD')
-    server = birdisle.Server(f'requirepass {password}')
-    yield server
-    server.close()
-
-
-@pytest.mark.unit
-@pytest.fixture(scope='session')
-def Birdisle(birdisle_server):
-    birdisle.redis.LocalSocketConnection.health_check_interval = 0
-    password = config.get_consumer_config().get('REDIS_PASSWORD')
-    r = birdisle.redis.StrictRedis(server=birdisle_server, password=password)
-    r.config_set('notify-keyspace-events', 'KEA')
-    return r
+def RedisInstance():
+    password = os.environ.get('REDIS_PASSWORD')
+    r = Redis(host='redis', password=password)
```

Comment:
will this work if no password is set?

Reply:
This fixture is just for testing, but currently no password is set on the test redis, so it should work as is.